### DISCUSSION TITLE
Higlighting affected genes on a pathway

### DISCUSSION THREAD

**Discussion by kathcand** [wikipathways/wikipathways-help#65] [Q&A] [answered]
Hi everyone! I have a list of dysregulated genes that enrich the MAPK signaling pathway (Mus musculus).
I want to show the Wikipathway figure in my paper with those dysregulated genes highlighted in a different color
I can edit the genes one by one on PathVisio but I was wondering if there is a way to do it by uploading my gene list and letting the software highlight them. Thanks in advance for your help!

**Comment by mkutmon** [comment, marked as answer]
Dear @kathcand!
Yes, you can do that in PathVisio or Cytoscape. You load your data (with a matching identifier) and then use the visualization tools to overlay the data.

We have tutorials for both tools with examples, so you can learn how to import your own data:
* For PathVisio, you can follow the hands-on example [here](https://pathvisio.org/tutorials/handson1.html). 
* For Cytoscape, you can use the WikiPathways App tutorial [here](https://cytoscape.org/cytoscape-tutorials/protocols/wikipathways-app/#/title).

**Reply by mkutmon** [reply]
If you just want to highlight a gene list that you have without any data, you could use the search bar in [Cytoscape](https://cytoscape.org/) to select the required nodes and then use the bypass option to change their color. I think that would be the easiest option that I can think of.

![image](https://github.com/wikipathways/wikipathways-help/assets/2158343/6f6b5fb5-f083-4c2d-a77b-89bc34c9a63b)
![image](https://github.com/wikipathways/wikipathways-help/assets/2158343/25f1e5af-e457-4a8e-8f1f-6fc69ac717c8)
![image](https://github.com/wikipathways/wikipathways-help/assets/2158343/fb3e89f7-0be7-44f1-a69d-eab4ab140193)
